tests/network/flat_overlay/conftest.py

```diff
@@ -44,14 +42,6 @@
 SPECIFIC_HOST_MASK = "32"
```

Comment:
Already exists in tests/network/flat_overlay/conftest.pym, please re-use (move it to a lower location).